.github/workflows/frontend.yml

```diff
@@ -14,6 +14,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: refs/heads/master
```

Comment:
This will check out master instead of the branch being tested, which isn't what we want. What we want is for the checkout action to clone enough history that it covers master as well, by default only the tip of the feature branch is cloned.

Reply:
Yeah, exactly @Rugvip ! Because of this, the test passes. Maybe, there's some kind of config to use below `- uses: actions / checkout @ v2` line.